plugins/inputs/riemann_listener/README.md

```diff
@@ -0,0 +1,41 @@
+# Riemann Listener Input Plugin
+
+The Riemann Listener is a simple input plugin that listens for messages from
+client that use riemann clients using riemann-protobuff format.
+
+
+### Configuration:
+
+This is a sample configuration for the plugin.
+
+```toml
+ ## URL to listen on
+ ## Default is "tcp://:5555"
```

Comment:
plugin name is missing here, and spacing is slightly off.

Reply:
Will modify this